Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+RUN apt-get update \
+    # Install gcc for x86 and ARM
+    && apt-get install -y build-essential gcc-arm-linux-gnueabi gcc-arm-linux-gnueabihf git \
```

Comment:
Please create separate images, one for x86 (don't include arm), and one for arm, etc.

Reply:
solved